### PR TITLE
Correct the Half Time mod description for osu!catch

### PR DESCRIPTION
**Pull request by Darius-Wattimena**
The description of the halftime mod for osu!catch isn't correct as it only reduces the speed of the beatmap and the catcher.

### CODE REVIEW

**Review by RedcXca — CHANGES_REQUESTED** [review]
For the trivia section I think the wording of "decreases the speed of beatmaps by 75%" can be misleading. I would rewrite it as "...misnomer, as the HT mod does not actually change the speed of a beatmap to 50% of the original but rather to 75%"

**Comment by Darius-Wattimena** [comment]
> For the trivia section I think the wording of "decreases the speed of beatmaps by 75%" can be misleading. I would rewrite it as "...misnomer, as the HT mod does not actually change the speed of a beatmap to 50% of the original but rather to 75%"

The current wording in the trivia section should be fine, also I rather want to keep this PR small to fix just the description of catch

File: wiki/Game_modifier/Half_Time/en.md [hunk 1]
```diff
@@ -38,9 +38,7 @@ As a result, using the Half Time mod will lead to an increase in maximum possibl
 
 ### osu!catch
 
-In [osu!catch](/wiki/Game_mode/osu!catch), the BPM is lowered by the same factor as in other modes. Though it *also* decreases the player character's speed so normal fruits without any mods *may* turn into hyperdash fruits.
-
-In addition to this, the leniency for hyperdashes is increased, which makes it much more difficult to stop underneath the next fruit when performing a hyperdash.
+In [osu!catch](/wiki/Game_mode/osu!catch), the BPM is lowered by the same factor as in other modes. Additionally, the speed of the catcher is also decreased. All the [fruits](/wiki/Hit_object/Fruit), [drops](/wiki/Hit_object/Juice_stream#drop), [droplets](/wiki/Hit_object/Juice_stream#droplet) and [bananas](/wiki/Hit_object/Banana) stay untouched.
```
Comment (MilkyIQ) [inline wiki/Game_modifier/Half_Time/en.md:41]:
```suggestion
In [osu!catch](/wiki/Game_mode/osu!catch), the BPM and the speed of the catcher is lowered by the same factor as in other modes. All the [fruits](/wiki/Hit_object/Fruit), [drops](/wiki/Hit_object/Juice_stream#drop), [droplets](/wiki/Hit_object/Juice_stream#droplet) and [bananas](/wiki/Hit_object/Banana) stay untouched.
```
sounds a bit better imo

Comment (TPGPL) [reply wiki/Game_modifier/Half_Time/en.md:41]:
```suggestion
In [osu!catch](/wiki/Game_mode/osu!catch), the BPM and the speed of the catcher are lowered by the same factor as in other modes. All the [fruits](/wiki/Hit_object/Fruit), [drops](/wiki/Hit_object/Juice_stream#drop), [droplets](/wiki/Hit_object/Juice_stream#droplet) and [bananas](/wiki/Hit_object/Banana) stay untouched.
```

**Review by TicClick — APPROVED** [review]

**Pull request merged**